skorch/tests/test_net.py

```diff
@@ -144,7 +145,18 @@ def test_net_learns(self, net_fit, data):
         y_pred = net_fit.predict(X)
         assert accuracy_score(y, y_pred) > 0.7
 
-    def test_predict_proba(self, net_fit, data):
+    def test_forward(self, net_fit, data):
+        X = data[0]
+        n = len(X)
+        y_forward = net_fit.forward(X)
+
+        assert is_torch_data_type(y_forward)
+        assert y_forward.shape == (n, 2)
```

Comment:
A comment what you expect would be nice, e.g.

    # Expecting (number of samples, number of output units)